### PR TITLE
Add mitakacloud7 cloudsource

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by jsuchome** [comment]
Could you add `mitakacloud7` value to mkcloud as well? (usage and sanity_checks)

**Comment by dirkmueller** [comment]
@jsuchome done

**Comment by jsuchome** [comment]
+1

**Comment by toabctl** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1281,6 +1281,11 @@ function onadmin_set_source_variables()
             CLOUDSLE12TESTISO="CLOUD-7-TESTING-$arch*Media1.iso"
             CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-7-devel"
         ;;
+        mitakacloud7)
+            CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/7:/Mitaka/images/iso
+            CLOUDSLE12DISTISO="SUSE-OPENSTACK-CLOUD-7-$arch*Media1.iso"
+            CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-7-official"
+        ;;
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: I don't think that's enough; you probably need to do changes in all other places where we mention develcloud7, and there are probably other locations where we check if the source starts with develcloud...

Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: Thats right, we treat it more like cloudsource =~ susecloud7 (e.g. we skip cct). I think thats okay..

**Pull request merged**